app/Config/bootstrap.php

```diff
@@ -22,6 +22,17 @@
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
 
+/**
+ * Load Composer autoload.
+ *
+ * Remove and re-prepend CakePHP's autoloader as Composer thinks it is the most important.
+ * @see http://goo.gl/kKVJO7
```

Comment:
i say use `https://github.com/composer/composer/commit/c80cb76` instead of short/indirect url `http://goo.gl/kKVJO7`